### ISSUE TITLE
improved discarding of message from mapping script

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eppye-bots/bots#10]
```
it is possible to discard a message form the mapping script.
but it is needed to write 'dummy data'; and this data has to be valid (for the 
outgoing message format).
thi is imporved: it is no longer needed to write the dummy data (btw, it is 
allowed to do this).

```

Original issue reported on code.google.com by `hjebb...@gmail.com` on 13 Jul 2010 at 1:22
- Blocked on: #20100712
